setup.py

```diff
@@ -157,30 +160,29 @@ def write_version(filename=os.path.join(*['airflow',
 datadog = ['datadog>=0.14.0']
 doc = [
     'mock',
-    'sphinx>=1.2.3',
     'sphinx-argparse>=0.1.13',
+    'Sphinx-PyPI-upload>=0.2.1',
     'sphinx-rtd-theme>=0.1.6',
+    'sphinx>=1.2.3',
     'sphinxcontrib-httpdomain>=1.7.0',
-    'Sphinx-PyPI-upload>=0.2.1'
 ]
 docker = ['docker~=3.0']
 druid = ['pydruid>=0.4.1']
 elasticsearch = [
+    'elasticsearch-dsl>=5.0.0,<6.0.0',
     'elasticsearch>=5.0.0,<6.0.0',
-    'elasticsearch-dsl>=5.0.0,<6.0.0'
 ]
-emr = ['boto3>=1.0.0, <1.8.0']
 gcp_api = [
-    'httplib2>=0.9.2',
     'google-api-python-client>=1.6.0, <2.0.0dev',
-    'google-auth>=1.0.0, <2.0.0dev',
     'google-auth-httplib2>=0.0.1',
-    'google-cloud-container>=0.1.1',
+    'google-auth>=1.0.0, <2.0.0dev',
     'google-cloud-bigtable==0.31.0',
+    'google-cloud-container>=0.1.1',
     'google-cloud-spanner>=1.7.1',
     'grpcio-gcp>=0.2.2',
+    'httplib2>=0.9.2',
```

Comment:
This appears to be a new dep here? Did you indent to include this change?

Reply:
This is not a new dependency. Previously, It was in the first position, but I now introduced the alphabetical order.